sdk/node/Libplanet.Node/Options/GenesisOptions.cs

```diff
@@ -17,10 +19,13 @@ public static readonly PrivateKey AppProtocolKey
         AppProtocolKey, 1);
 
     [PrivateKey]
+    [Description("The key of the genesis block.")]
     public string GenesisKey { get; set; } = string.Empty;
 
     [PublicKeyArray]
+    [Description("The hash of the genesis block.")]
```

Comment:
This is not a `hash of the genesis block`.

Reply:
> This is not a `hash of the genesis block`.

amended https://github.com/planetarium/libplanet/pull/3919/commits/db6882cd8f42dec38b99a27e8505cd2d06ecd8bd